lib/db.c

```diff
@@ -15230,6 +15245,10 @@ grn_db_recover_database_remove_orphan_inspect(grn_ctx *ctx, grn_obj *db)
 static void
 grn_db_recover_database(grn_ctx *ctx, grn_obj *db)
 {
+  if (grn_db_recover_skip_target(ctx, db, target_last_modified)) {
+    GRN_LOG(ctx, GRN_LOG_INFO, "[db][recover] skipped to recover: <%s>", grn_obj_get_io(ctx, db)->path);
```

Comment:
In my opinion, we should write recover target in a log as below.
 
`"[db][recover] skipped to recover database: <%s>"`

Reply:
fixed.